client/my-sites/plans-features-main/index.tsx

```diff
@@ -573,6 +573,13 @@ const PlansFeaturesMain = ( {
 
 	const isLoadingGridPlans = Boolean( intentFromSiteMeta.processing || ! gridPlans );
 
+	const comparisonGridContainerClasses = classNames(
+		'plans-features-main__comparison-grid-container',
+		{
+			isHidden: ! showPlansComparisonGrid,
```

Comment:
Nit. case of css class `is-hidden`

Reply:
Hmm yeah, good call. It should be `is-hidden`